core/jobs/job_utils.py

```diff
@@ -102,9 +102,10 @@ def get_model_class(kind: Optional[str]) -> Type[datastore_services.Model]:
     # separate workers and some parts of the jobs are probably not available
     # to all the workers.
     models.Registry.get_all_storage_model_classes()
-    return datastore_services.Model._lookup_model( # pylint: disable=protected-access
-        kind
-    )
+
+    model_instance = datastore_services.Model._lookup_model(kind) # pylint: disable=protected-access
+    assert model_instance is not None, f'Model with kind "{kind}" not found.'
+    return type(model_instance)
```

Comment:
Why do we return type(model_instance) rather than model_instance? Does this change what the previous code does? (Also I am not sure, but does _lookup_model actually return a class or an instance?)